doc/manual.md

```diff
@@ -3060,6 +3060,30 @@ But these will compile successfully:
   doAssert string(s) == "test"
   ```
 
+If a procedure call returns a `var`-typed value and is assigned to a `var`-declared variable, it is copied. This way, you can modify any `var`-declared variable and be sure it will not affect variables out of its scope.
+
+By contrast, passing a `var`-typed return value to another procedure call does not copy it. See [Var-parameters](#procedures-var-parameters)
+
+For a `var`-declared variable that *can* modify an existing `var`-typed value, the `addr` procedure can be used to create a reference type, see [Reference and pointer types](#types-reference-and-pointer-types)
+
+
+```nim
+var foo = Foo(x: "this")
+
+proc getX(f: var Foo): var string
+  f.x
+
+var x = foo.getX         # puts a copy of foo.x into x
+x = "that"               # changes the copy
+echo foo.x               # prints "this", the original value is unchanged
+
+var z = addr foo.getX    # puts a reference to foo.x into x
```

Comment:
In the comment I think you meant to say "into z"

Reply:
I'd also not say "a reference" as it's not a `ref` but a `ptr` so  something along "stores pointer to foo.x" is more clear.